src/i18n/RuleEngine.js

```diff
@@ -600,8 +595,8 @@ NOTE: Non-shared subscription will not work well when EMQX is clustered.`,
     en: "For example: ${'{'}payload{'}'}, ${'{'}clientid{'}'}, ${'{'}topic{'}'} , ${'{'}username{'}'}, etc. Use fields according to the data bridges requirements of your business and forwards the message as it is if it is empty.",
   },
   ingressRemoteTopicDesc: {
-    zh: '本地服务将订阅该远程服务的主题接收消息。',
-    en: 'The local broker will subscribe to the remote broker topic to receive messages.',
+    zh: '本地服务将订阅该远程服务的主题接收消息。<br/>当 EMQX 配置为集群或启用了 ingress 连接池时，**必须**使用共享订阅来避免消息重复。',
+    en: 'The local broker will subscribe to the remote broker topic to receive messages. <br/>When EMQX is running in a cluster or with an enabled ingress connection pool, it is **mandatory** to use shared subscriptions to avoid message duplication.',
```

Comment:
```suggestion
    en: 'Topic subscribed by the local broker from the remote broker to receive messages. <br/>When EMQX is running in a cluster or with an enabled ingress connection pool, it is **mandatory** to use shared subscriptions to avoid message duplication.',
```